scripts/run.sh

```diff
@@ -13,10 +13,9 @@ fi
 
 set -x
 
-docker run \
--v $RWD_DATA:/opt/rwd-data \
--p 5000:5000 \
+docker-compose run \
 --rm \
-$ARGS \
-docker-rwd \
---reload
+-e RWD_DATA=$RWD_DATA \
+--name docker-rwd \
```

Comment:
They're not required...wasn't sure if we had added that for convenience. Without the flag ends up with the name ` rapidwatersheddelineation_app` which is pretty good.